### PR TITLE
ci(deploy): update site-deploy.yml

### PR DESCRIPTION
**Pull request by Wxh16144**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE_CN.md?plain=1)

### 🤔 This is a ...

- [ ] 🆕 New feature
- [ ] 🐞 Bug fix
- [ ] 📝 Site / documentation improvement
- [ ] 📽️ Demo improvement
- [ ] 💄 Component style improvement
- [ ] 🤖 TypeScript definition improvement
- [ ] 📦 Bundle size optimization
- [ ] ⚡️ Performance optimization
- [ ] ⭐️ Feature enhancement
- [ ] 🌐 Internationalization
- [ ] 🛠 Refactoring
- [ ] 🎨 Code style optimization
- [ ] ✅ Test Case
- [ ] 🔀 Branch merge
- [x] ⏩ Workflow
- [ ] ❓ Other (about what?)

### 🔗 Related Issues

ref: https://github.com/ant-design/ant-design/commit/5533b037a78be8144c7518e4d2527f48b9b35118

> - Describe the source of related requirements, such as links to relevant issue discussions.
> - For example: close #xxxx, fix #xxxx

### 💡 Background and Solution

> - The specific problem to be addressed.
> - List the final API implementation and usage if needed.
> - If there are UI/interaction changes, consider providing screenshots or GIFs.

### 📝 Change Log

> - Describe the impact of the changes on developers, not the solution approach.
> - Reference: https://ant.design/changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |      --     |
| 🇨🇳 Chinese |     --      |

### CODE REVIEW

File: .github/workflows/site-deploy.yml [hunk 1]
```diff
@@ -82,13 +82,23 @@ jobs:
...
+          echo "include: [/_*.*]" >> _site/_config.yml
+
       - name: Deploy to GitHub Pages
         uses: peaceiris/actions-gh-pages@v4
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           publish_dir: ./_site
           force_orphan: true
 
+      - name: Remove _config.yml (temporary)
```
Comment (zombieJ) [inline .github/workflows/site-deploy.yml:98]: `_site` 本来就是临时文件夹，是不是不用删的？

Comment (Wxh16144) [reply .github/workflows/site-deploy.yml:98]: `rm _site/_config.yml`  把这个只是针对 GitHub Pages 的配置删除了， 担心后续的部署 Surge 被影响到了

**Review by zombieJ — APPROVED** [review]

File: .github/workflows/site-deploy.yml (hunk 1 above)
Comment (zombieJ) [reply .github/workflows/site-deploy.yml:98]: 细👍~

Comment (afc163) [reply .github/workflows/site-deploy.yml:98]: 等 mako 来解更好：https://github.com/umijs/mako/issues/1492

Comment (zombieJ) [reply .github/workflows/site-deploy.yml:98]: > 等 mako 来解更好：https://github.com/umijs/mako/issues/1492

👍 这周发包赶得上不~~

**Pull request closed without merging**